eth/contracts/facets/DFAdminFacet.sol

```diff
@@ -24,7 +24,7 @@ contract DFAdminFacet is WithStorage {
     /////////////////////////////
 
     modifier onlyAdmin() {
-        LibDiamond.enforceIsContractOwner();
+        LibDiamondHelper.enforceIsContractOwner();
```

Comment:
LibPermissions sg to me. My only thought is that if we extend the Lib to have more diamond functionalities, it might make sense to keep `Diamond` in the name. But, we can just make another lib that's specific to `DiamondBaseStorage` in that case. So I'll update the name.

Reply:
Or `LibAccess` ?